toolkit/tools/pkg/imagecustomizerlib/testdata/verity-usr-config.yaml

```diff
@@ -24,7 +24,7 @@ storage:
     name: usr
     dataDeviceId: usr
     hashDeviceId: usrhash
-    corruptionOption: panic
+    corruptionOption: io-error
```

Comment:
Does `io-error` has specific meaning for usr verity?

Reply:
No, just changing one of the verity configs so that the different tests can check for different values.